README.md

```diff
@@ -1,7 +1,25 @@
-# sidecar-proxy
-A reverse proxy, meant to run in a container alongside a service it needs to serve (in the same pod).
+# Sidecar proxy
+A container meant to run alongside a service (in the same pod) in order to serve Prometheus metrics for that service.
 
-Supports both HTTP (Go's built in ReverseProxy) and Websocket (https://github.com/koding/websocketproxy) requests.
-In addition, serves Prometheus type metrics through the `/metrics` endpoint.
+Supported metrics:
```

Comment:
Suggest to add subblocks here per supported services, otherwise jupyter_kernel_busyness looks really weird as a supported metric since it's very service specific